core/domain/expression_parser.py

```diff
@@ -78,9 +78,9 @@ def contains_balanced_brackets(expression):
     return len(stack) == 0
 
 
-def is_algebraic(expression):
-    """Checks if the given expression is algebraic. An algebraic expression must
-    contain at least one valid identifier (latin letter or greek symbol name).
+def contains_variables(expression):
+    """Checks if the given expression contains at least
+    one valid identifier (latin letter or greek symbol name).
```

Comment:
Maybe move words from this line to the previous one so that the latter extends roughly up to the line limit.

Reply:
Done.